test/unit/file_transfer_agent/encrypt_util_test.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2021 Snowflake Computing Inc. All rights reserved.
+ */
+
+var assert = require('assert');
+var mock = require('mock-require');
+var SnowflakeEncryptionUtil = require('./../../../lib/file_transfer_agent/encrypt_util').encrypt_util;
+
+describe('Encryption util', function ()
+{
+  var encryptionMaterial;
+  var mockData = 'mockData';
+  var mockFileName = 'mockFileName';
+  var mockRandomBytes = 'mockRandomBytes';
+  var mockTmpDir = 'mockTmpDir';
+  var mockTmpName = 'mockTmpName';
+
+  var EncryptionUtil;
+  var encrypt;
+  var filestream;
+  var temp;
+
+  this.beforeEach(function ()
+  {
+    encryptionMaterial = {
+      queryStageMasterKey: 'ztke8tIdVt1zmlQIZm0BMA==',
```

Comment:
Is this a valid queryStageMasterKey or a dummy one that is just used for testing?

Reply:
It's based from a python test which I believe is a dummy: https://github.com/snowflakedb/snowflake-connector-python/blob/6cec12333927ac4f85bc0c9317dde7baf7097364/test/unit/test_encryption_util.py#L24